request-manager/resources/request_manager_schema.sql

```diff
@@ -41,3 +42,14 @@ CREATE TABLE IF NOT EXISTS `job_log` (
 
   PRIMARY KEY (`request_id`, `job_id`, `try`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+CREATE TABLE IF NOT EXISTS `suspended_job_chains` (
+  `request_id`          BINARY(20) NOT NULL,
+  `suspended_job_chain` LONGBLOB NOT NULL,
+  `rm_host`             VARCHAR(64) NULL DEFAULT NULL,
```

Comment:
Why is the Request Manager host saved?

Reply:
Setting the RM host is how the resumer claims an SJC - it sets rm_host from null to its hostname.